zgw/zaakdetails/src/main/kotlin/com/ritense/zaakdetails/documentobjectenapisync/DocumentObjectenApiSyncService.kt

```diff
@@ -72,7 +77,14 @@ class DocumentObjectenApiSyncService(
         documentObjectenApiSyncRepository.save(modifiedSync)
     }
 
-    fun deleteSyncConfigurationByDocumentDefinition(documentDefinitionName: String, documentDefinitionVersion: Long) {
+    fun deleteSyncConfigurationByDocumentDefinition(
+        @LoggableResource(resourceType = JsonSchemaDocumentDefinition::class) documentDefinitionName: String,
+        documentDefinitionVersion: Long
+    ) {
+        logger.info {
```

Comment:
Why with the + in between? Something like this would seem more readable to me: 
logger.info { 
    "Deleting sync configuration: documentDefinitionName=$documentDefinitionName, documentDefinitionVersion=$documentDefinitionVersion" 
}

Reply:
triple double quotes is the only alternative. Will do that